tf2onnx/function/gathernd.py

```diff
@@ -43,19 +43,16 @@ def make_gathernd_inner_loop(ctx, params, index, dtype):
     inner_loop = ctx.make_node("Loop", [trip_node.output[0],
                                         cond_const.output[0],
                                         params],
+                               name=name,
                                attr={"body": body_graph})
     nodes.append(inner_loop.op)
     return nodes, inner_loop
 
-def gathernd_op(ctx, node, name, args):
-    """GatherNd op."""
-    # T output = GatherNd(T Input, INT32/INT64 indices)
+
+def make_gathernd_subgraph(ctx, params, indices, output, name, t_params):
```

Comment:
from make_gathernd perspective, there is no name concept, but scope/prefix, right?
change it to scope_name?

Reply:
so are the others.